------------------------------------------------------------------------
r84968 | danieljames | 2013-07-07 12:46:48 +0100 (Sun, 07 Jul 2013) | 5 lines

Thread: rollback change in #8070 as this includes a regression on windows XP.

From [84946], by viboes.


------------------------------------------------------------------------
--- 1_54_0/boost/thread/win32/thread_primitives.hpp	(revision 84967)
+++ 1_54_0/boost/thread/win32/thread_primitives.hpp	(revision 84968)
@@ -19,7 +19,7 @@
 
 #ifndef BOOST_THREAD_WIN32_HAS_GET_TICK_COUNT_64
 #if _WIN32_WINNT >= 0x0600
-#define BOOST_THREAD_WIN32_HAS_GET_TICK_COUNT_64
+//#define BOOST_THREAD_WIN32_HAS_GET_TICK_COUNT_64
 #endif
 #endif
 
